sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedFluxBase.java

```diff
@@ -38,15 +37,11 @@
  * @see Page
  * @see Flux
  */
-public class PagedFluxBase<T, P extends PagedResponse<T>> extends Flux<T> {
-
-    private final Supplier<Mono<P>> firstPageRetriever;
-
-    private final Function<String, Mono<P>> nextPageRetriever;
-
+public class PagedFluxBase<T, P extends PagedResponse<T>> extends ContinuablePagedFluxCore<String, T, P> {
```

Comment:
It might be worth considering if this entire class should be deprecated now. I understand it is still the parent class of PagedFlux, but that is only while we don't make breaking changes.

Reply:
yes, both `PagedFlux` and `PagedFluxBase` has exact same code in it. As you pointed the reason I didn't add deprecation tag is `PagedFluxBase` is base type for the "Stable" type `PagedFlux`.

Reply:
@JonathanGiles is it still ok to have deprecated tag for PagedFluxBase? If so we can add it and recommend using `ContinuablePagedFluxCore`.

Reply:
Marked as deprecated with suggestion to use ContinuablePagedFlux instead.